src/niveristand/clientapi/_stimulusprofilesession.py

```diff
@@ -52,7 +52,7 @@ def deploy(self, auto_start):
             str: The ID of the Stimulus profile session.
 
         """
-        ret_val, session_id, err = self._dot_net_instance.Deploy(auto_start, None, None)
+        session_id, err = self._dot_net_instance.Deploy(auto_start, None, None)
```

Comment:
This should resolve Issue #50, [ValueError Exception when deploying .CSV as RT Sequence](https://github.com/ni/niveristand-python/issues/50).